.github/workflows/tests.yml

```diff
@@ -3,34 +3,96 @@ name: Tests
 on:
   push:
     branches:
-      - 1.0
-      - 1.1
-      - 1.2
+      - '1.2'
```

Comment:
Are we no longer testing on the other two branches?

Reply:
The other two branches will use the [old `tests.yml` config](https://github.com/wintercms/winter/blob/1.1/.github/workflows/tests.yml), which runs all the tests within the `tests` folder.

I've split off this config to run only 1.2 and develop as we moved all the tests into the System module.